gamemode/modules/hungermod/sv_hungermod.lua

```diff
@@ -20,7 +20,7 @@ local function HMAFKHook(ply, afk)
     if afk then
         ply.preAFKHunger = ply:getDarkRPVar("Energy")
     else
-        ply:setDarkRPVar("Energy", ply.preAFKHunger or 100)
+        ply:setSelfDarkRPVar("Energy", ply.preAFKHunger or 100)
```

Comment:
Why this change? All other places use `setSelfDarkRPVar` for Energy

Reply:
I know, I replaced it with setSelfDarkRPVar

Reply:
oh, whoops